app/src/main/java/com/babylon/wallet/android/presentation/wallet/WalletScreen.kt

```diff
@@ -285,14 +285,14 @@ private fun RadixBanner(
                 )
 
                 Text(
-                    text = stringResource(id = R.string.homePage_radixBanner_title),
+                    text = "", // stringResource(id = R.string.homePage_radixBanner_title),
```

Comment:
Disregard this for now, breaking changes in main so had to comment out in order to compile project.